docs/docs/general/02-Components/AskUI-Development-Environment.md

```diff
@@ -185,3 +186,74 @@ Note: Adjust the parameters as needed for your specific configuration.
 :::
 
 See also the dedicated [AskUI Runner docs](AskUI-Runner.md) for more information.
+
+
+## ADE + VSCode IDE
+
+The ADE is build to work seamlessly with (VSCode)[https://code.visualstudio.com/] together. Therefore we create (VScode Workspace Settings)[https://code.visualstudio.com/docs/getstarted/settings#_settingsjson]
```

Comment:
To me as a user I think it would be unclear if I am supposed to create these settings or if they are created for me.

Reply:
I've rewrote it. Can you cross check?

Reply:
Would be clear now ✅